### PR TITLE
migrate mock-handling to tiger

### PR DESCRIPTION
**Pull request by ivanmjartan**
---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
// Tiger platform
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

```
// Bear platform
extended test - cypress - integrated
extended test - cypress - isolated
extended test - cypress - record
```

### CODE REVIEW

**Comment by ivanmjartan** [comment]
extended test - backstop

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18402/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20040/ : FAILURE in 15m 14s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2606/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18404/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/2075/ : UNSTABLE in 30m 23s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18410/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20046/ : FAILURE in 11m 16s

**Comment by ivanmjartan** [comment]
extended test - backstop

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18414/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20051/ : FAILURE in 12m 21s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2607/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18415/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/2077/ : UNSTABLE in 29m 30s

**Comment by ivanmjartan** [comment]
extended test - backstop

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18417/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20053/ : FAILURE in 11m 17s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2608/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18419/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20054/ : FAILURE in 11m 21s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18420/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20055/ : FAILURE in 6m 04s

**Comment by ivanmjartan** [comment]
ok to test

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18422/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20057/ : FAILURE in 10m 42s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18427/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20065/ : FAILURE in 6m 52s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18428/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20066/ : FAILURE in 11m 04s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18429/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20067/ : FAILURE in 11m 05s

**Comment by ivanmjartan** [comment]
extended test - backstop

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2609/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18430/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/2079/ : UNSTABLE in 29m 24s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18433/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20072/ : FAILURE in 11m 52s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18436/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20077/ : FAILURE in 14m 14s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18438/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20079/ : FAILURE in 10m 58s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18443/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20084/ : FAILURE in 11m 13s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18444/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20085/ : FAILURE in 6m 14s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18447/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20088/ : FAILURE in 11m 03s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18448/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20090/ : FAILURE in 10m 51s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18454/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20096/ : FAILURE in 10m 49s

**Comment by ivanmjartan** [comment]
extended test - backstop

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18455/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20097/ : FAILURE in 12m 45s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2610/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18456/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/2080/ : UNSTABLE in 30m 33s

**Comment by ivanmjartan** [comment]
extended test - backstop

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18459/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20101/ : SUCCESS in 10m 49s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2611/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18460/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/2081/ : SUCCESS in 24m 46s

File: libs/sdk-backend-base/src/normalizingBackend/tests/normalizer.test.ts [hunk 1]
```diff
@@ -256,7 +256,8 @@ describe("Normalizer", () => {
     });
 });
 
-describe("Denormalizer", () => {
+describe.skip("Denormalizer", () => {
+    // TODO this is skip due to hardcoded values in the test
```
Comment (kandl) [inline libs/sdk-backend-base/src/normalizingBackend/tests/normalizer.test.ts:260]: Should we add reference to some ticket, so we can easily found all the remaining parts that we are going to migrate later?

File: libs/sdk-backend-mockingbird/src/recordedBackend/tests/execution.test.ts [hunk 2]
```diff
@@ -26,12 +26,6 @@ describe("recordedDataView", () => {
         expect(dv.result).toBeDefined();
     });
 
-    it("should load result with uriRefs by default", () => {
-        const dv = recordedDataView(ReferenceRecordings.Scenarios.BarChart.SingleMeasureWithViewBy);
-
-        assertExpectedRefs(dv.result, isUriRef);
```
Comment (kandl) [inline libs/sdk-backend-mockingbird/src/recordedBackend/tests/execution.test.ts:32]: Should we change the default to idRefs, and just update the tests (assert isIdRef)?

File: libs/sdk-backend-mockingbird/src/recordedBackend/tests/execution.test.ts [hunk 3]
```diff
@@ -44,18 +38,6 @@ describe("recordedDataView", () => {
...
-    it("should load result with uriRefs by default", async () => {
-        const result = await recordedBackend(ReferenceRecordings.Recordings)
-            .workspace("reference-workspace")
-            .execution()
-            .forDefinition(
-                ReferenceRecordings.Scenarios.BarChart.SingleMeasureWithViewBy.execution.definition,
-            )
-            .execute();
-
-        assertExpectedRefs(result, isUriRef);
```
Comment (kandl) [inline libs/sdk-backend-mockingbird/src/recordedBackend/tests/execution.test.ts:56]: Should we change the default to idRefs, and just update the tests (assert isIdRef)?

File: libs/sdk-ui-charts/src/highcharts/chartTypes/waterfallChart/tests/waterfallChartOptions.test.ts [hunk 4]
```diff
@@ -123,41 +119,10 @@ describe("waterfallChartOptions", () => {
 
             expect(seriesData[seriesData.length - 1].isSum).not.toBeDefined();
         });
-
-        it("should add the shadow colum for the metric is total", () => {
```
Comment (kandl) [inline libs/sdk-ui-charts/src/highcharts/chartTypes/waterfallChart/tests/waterfallChartOptions.test.ts:127]: Should we rather skip the removed tests and migrate later?

File: libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/tests/saveAsDashboardHandler.test.ts [hunk 5]
```diff
@@ -130,23 +124,18 @@ describe("save as dashboard handler", () => {
         beforeEach(async () => {
             await preloadedTesterFactory((tester) => {
                 Tester = tester;
-            }, LockedDashboardIdentifier);
+            }, SimpleDashboardIdentifier);
         });
 
         const TestDashboardTitle = "My Dashboard Copy";
 
-        it("should save an existing locked dashboard as a copy with isLocked flag cleaned", async () => {
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/tests/saveAsDashboardHandler.test.ts:138]: Should we skip and migrate later?

File: libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/tests/saveDashboardHandler.test.ts [hunk 6]
```diff
@@ -135,32 +135,4 @@ describe("getDashboardWithSharing", () => {
             expect(isUnderStrictControl).toBe(expectedResult.isUnderStrictControl);
         },
     );
-
-    it.each([
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/tests/saveDashboardHandler.test.ts:139]: Should we skip and migrate later?

File: libs/sdk-ui-dashboard/src/model/commandHandlers/drill/common/test/filterValuesResolver.test.ts [hunk 7]
```diff
@@ -32,17 +32,6 @@ describe("resolveFilterValues", () => {
         expect(result).toMatchSnapshot();
     });
 
-    it("should return resolved attribute filters elements by ref", async () => {
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/drill/common/test/filterValuesResolver.test.ts:35]: Skip and migrate later?

File: libs/sdk-ui-dashboard/src/model/commandHandlers/drill/test/drillToDashboardHandler.test.ts [hunk 8]
```diff
@@ -1,80 +0,0 @@
-// (C) 2023 GoodData Corporation
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/drill/test/drillToDashboardHandler.test.ts:1]: Should we only skip and migrate later?

File: libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/attributeFilter/tests/removeAttributeFilterHandler.test.ts [hunk 9]
```diff
@@ -44,7 +42,6 @@ describe("removeAttributeFilterHandler", () => {
         const displayForms = selectAttributeFilterDisplayFormsMap(Tester.state());
         const remainingFilters = selectFilterContextAttributeFilters(Tester.state());
 
-        expect(displayForms.size).toEqual(remainingFilters.length);
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/attributeFilter/tests/removeAttributeFilterHandler.test.ts:47]: Why this needs to be removed?

File: libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/attributeFilter/tests/removeAttributeFilterHandler.test.ts [hunk 10]
```diff
@@ -84,46 +81,4 @@ describe("removeAttributeFilterHandler", () => {
 
         expect(selectFilterContextAttributeFilters(Tester.state())).toEqual(originalFilters);
     });
-
-    it("should remove the filter from widget ignore lists", async () => {
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/attributeFilter/tests/removeAttributeFilterHandler.test.ts:88]: Skip and migrate later?

File: libs/sdk-ui-dashboard/src/model/commandHandlers/layout/tests/addLayoutSectionHandler.test.ts [hunk 11]
```diff
@@ -267,19 +267,18 @@ describe("add layout section handler", () => {
...
-            const middleSectionAdded: DashboardLayoutSectionAdded = await Tester.dispatchAndWaitFor(
+
+            await Tester.dispatchAndWaitFor(
                 addLayoutSection(1, undefined, [TestInsightPlaceholderItem]),
                 "GDC.DASH/EVT.FLUID_LAYOUT.SECTION_ADDED",
             );
 
             const layout = selectLayout(Tester.state());
             expect(layout.sections.length).toBe(originalLayout.sections.length + 2);
-            expect(layout.sections[4]).toEqual(lastSectionAdded.payload.section);
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/layout/tests/addLayoutSectionHandler.test.ts:281]: Why this removal?

File: libs/sdk-ui-dashboard/src/model/commandHandlers/layout/tests/moveLayoutSectionHandler.test.ts [hunk 12]
```diff
@@ -35,16 +35,6 @@ describe("move section command handler", () => {
             expect(event.payload.reason).toEqual("USER_ERROR");
             expect(event.correlationId).toEqual(TestCorrelation);
         });
-
-        it("should fail if no move would happen", async () => {
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/layout/tests/moveLayoutSectionHandler.test.ts:39]: Why removed?

File: libs/sdk-ui-dashboard/src/model/commandHandlers/layout/tests/moveSectionItemHandler.test.ts [hunk 13]
```diff
@@ -25,16 +25,6 @@ describe("move layout section item handler", () => {
             }, SimpleDashboardIdentifier);
         });
 
-        it("should fail if bad source section index is provided", async () => {
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/layout/tests/moveSectionItemHandler.test.ts:28]: Why removed?

File: libs/sdk-ui-dashboard/src/model/commandHandlers/layout/tests/resizeHeightHandler.test.ts [hunk 14]
```diff
@@ -44,31 +43,6 @@ describe("resize section items height handler", () => {
     });
 
     describe("for valid indexes", () => {
-        it("should set height in items by index", async () => {
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/layout/tests/resizeHeightHandler.test.ts:47]: Again, why it's removed?

File: libs/sdk-ui-dashboard/src/model/commandHandlers/layout/tests/resizeWidthHandler.test.ts [hunk 15]
```diff
@@ -43,34 +42,6 @@ describe("resize section items width handler", () => {
     });
 
     describe("for valid indexes", () => {
-        it("should set width in items by index", async () => {
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/layout/tests/resizeWidthHandler.test.ts:46]: why remove and not skip?

File: libs/sdk-ui-dashboard/src/model/commandHandlers/widgets/tests/changeInsightWidgetFilterSettingsHandler.test.ts [hunk 16]
```diff
@@ -1,465 +0,0 @@
...
-                ),
-                "GDC.DASH/EVT.INSIGHT_WIDGET.FILTER_SETTINGS_CHANGED",
-            );
-
-            expect(event.payload.ignoredAttributeFilters).toEqual([]);
-
-            expect(ignoredWidgetFilterRefs(Tester, WidgetWithNoFilters.ref)).toEqual([]);
-        });
-    });
-});
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/widgets/tests/changeInsightWidgetFilterSettingsHandler.test.ts:465]: Why remove this file? Why not skip it and migrate later?

File: libs/sdk-backend-mockingbird/api/sdk-backend-mockingbird.api.md [hunk 17]
```diff
@@ -71,7 +70,6 @@ export type CompositeBackendPart = {
 // @internal (undocumented)
 export type DashboardRecording = {
     obj: IDashboardWithReferences;
-    alerts: IWidgetAlert[];
```
Comment (kandl) [inline libs/sdk-backend-mockingbird/api/sdk-backend-mockingbird.api.md:74]: Why total removal of the support for alerting? (I think it's planned for Q2)

File: tools/mock-handling/src/index.ts [hunk 18]
```diff
@@ -29,7 +29,7 @@ program
     .version(LIB_VERSION)
     .option("--recordingDir <path>", "Directory with recording inputs and outputs")
     .option("--project-id <id>", "Project id from which you want to capture mock data")
-    .option("--username <email>", "Your username that you use to log in to GoodData platform.")
+    .option("--tigerToken <token>", "GoodData tiger platform auth token.")
     .option("--hostname <url>", `Instance of GoodData platform. The default is ${DEFAULT_HOSTNAME}`)
     .option("--config <path>", `Custom config file (default ${DEFAULT_CONFIG_FILE_NAME})`)
     .option("--backend <type>", `Backend (default ${DEFAULT_BACKEND})`)
```
Comment (kandl) [inline tools/mock-handling/src/index.ts:35]: I expect backend option should be removed?

File: tools/mock-handling/src/recordings/common.ts [hunk 19]
```diff
@@ -82,15 +82,19 @@ export function toJsonString(obj: object, options: WriteJsonOptions = {}): strin
     }
 
     if (replaceString) {
-        result = result.replace(new RegExp(replaceString[0], "g"), replaceString[1]);
+        replaceString.forEach(([searchString, replacement]) => {
+            result = result.replace(new RegExp(searchString, "g"), replacement);
+        });
+
+        //result = result.replace(new RegExp(replaceString[0], "g"), replaceString[1]);
```
Comment (kandl) [inline tools/mock-handling/src/recordings/common.ts:89]: Forgotten commented code

File: tools/mock-handling/src/recordings/dashboards.ts [hunk 20]
```diff
@@ -76,20 +73,22 @@ export class DashboardRecording implements IRecording {
 
         const ref = idRef(this.dashboardId);
 
-        const [dashboardWithReferences, alerts] = await Promise.all([
+        const [dashboardWithReferences] = await Promise.all([
             backend.workspace(workspace).dashboards().getDashboardWithReferences(ref),
-            backend.workspace(workspace).dashboards().getDashboardWidgetAlertsForCurrentUser(ref),
```
Comment (kandl) [inline tools/mock-handling/src/recordings/dashboards.ts:81]: I would rather just comment the code related to alerting as alerting support is coming to panther soon.

File: libs/sdk-ui-tests/scenarios/charts/barChart/base.tsx [hunk 21]
```diff
@@ -23,74 +23,166 @@ export const BarChartWithTwoMeasuresAndViewBy = {
...
 export const BarChartWithSingleMeasureAndTwoViewByAndStack = {
     measures: [ReferenceMd.Amount],
-    viewBy: [ReferenceMd.Product.Name, ReferenceMd.Region],
-    stackBy: ReferenceMd.Department,
+    viewBy: [ReferenceMd.Product.Name, ReferenceMd.Region.Default],
+    stackBy: ReferenceMd.Department.Default,
 };
 
+/*
+const arrayOfAccountValues = [
```
Comment (kandl) [inline libs/sdk-ui-tests/scenarios/charts/barChart/base.tsx:51]: Why this comment?

File: libs/sdk-ui-tests/scenarios/pivotTable/totals.tsx [hunk 22]
```diff
@@ -33,13 +33,13 @@ export const PivotTableWithTwoMeasuresGrandTotalsAndSubtotals = {
         newTotal("sum", ReferenceMd.Amount, ReferenceMd.Product.Name),
         newTotal("min", ReferenceMd.Amount, ReferenceMd.Product.Name),
         newTotal("max", ReferenceMd.Won, ReferenceMd.Product.Name),
-        newTotal("nat", ReferenceMd.Won, ReferenceMd.Product.Name),
-        newTotal("sum", ReferenceMd.Amount, ReferenceMd.Department),
-        newTotal("med", ReferenceMd.Amount, ReferenceMd.Department),
-        newTotal("med", ReferenceMd.Won, ReferenceMd.Department),
-        newTotal("nat", ReferenceMd.Won, ReferenceMd.Department),
+        // newTotal("nat", ReferenceMd.Won, ReferenceMd.Product.Name),
```
Comment (kandl) [inline libs/sdk-ui-tests/scenarios/pivotTable/totals.tsx:36]: Not sure about this commented code?

File: libs/sdk-ui-tests/stories/visual-regression/extensionComponents/attributeHierarchyDialog/AttributeHierarchyDialog.tsx [hunk 23]
```diff
@@ -1,122 +0,0 @@
-// (C) 2023 GoodData Corporation
```
Comment (kandl) [inline libs/sdk-ui-tests/stories/visual-regression/extensionComponents/attributeHierarchyDialog/AttributeHierarchyDialog.tsx:1]: Why this removal?

File: libs/sdk-backend-mockingbird/src/recordedBackend/tests/execution.test.ts (hunk 3 above)
Comment (ivanmjartan) [reply libs/sdk-backend-mockingbird/src/recordedBackend/tests/execution.test.ts:56]: Same as before test it uri by default thats why test deleted

File: libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/tests/saveAsDashboardHandler.test.ts (hunk 5 above)
Comment (ivanmjartan) [reply libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/tests/saveAsDashboardHandler.test.ts:138]: Locked is not supported in this way ... on tiger is done via parent workspace thats why deleted

File: libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/tests/saveDashboardHandler.test.ts (hunk 6 above)
Comment (ivanmjartan) [reply libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/tests/saveDashboardHandler.test.ts:139]: Locked is not supported in this way ... on tiger is done via parent workspace thats why deleted

File: libs/sdk-ui-dashboard/src/model/commandHandlers/drill/common/test/filterValuesResolver.test.ts (hunk 7 above)
Comment (ivanmjartan) [reply libs/sdk-ui-dashboard/src/model/commandHandlers/drill/common/test/filterValuesResolver.test.ts:35]: Again ref on tiger

File: libs/sdk-ui-dashboard/src/model/commandHandlers/drill/test/drillToDashboardHandler.test.ts (hunk 8 above)
Comment (ivanmjartan) [reply libs/sdk-ui-dashboard/src/model/commandHandlers/drill/test/drillToDashboardHandler.test.ts:1]: I guess not

File: libs/sdk-backend-mockingbird/api/sdk-backend-mockingbird.api.md (hunk 17 above)
Comment (ivanmjartan) [reply libs/sdk-backend-mockingbird/api/sdk-backend-mockingbird.api.md:74]: Tiger is not supporting alerts

File: libs/sdk-ui-tests/stories/visual-regression/extensionComponents/attributeHierarchyDialog/AttributeHierarchyDialog.tsx (hunk 23 above)
Comment (ivanmjartan) [reply libs/sdk-ui-tests/stories/visual-regression/extensionComponents/attributeHierarchyDialog/AttributeHierarchyDialog.tsx:1]: Again very complex hardcoded mock

File: libs/sdk-ui-tests/scenarios/charts/barChart/base.tsx (hunk 21 above)
Comment (ivanmjartan) [reply libs/sdk-ui-tests/scenarios/charts/barChart/base.tsx:51]: removed

File: tools/mock-handling/src/recordings/dashboards.ts (hunk 20 above)
Comment (ivanmjartan) [reply tools/mock-handling/src/recordings/dashboards.ts:81]: Same not supported

File: libs/sdk-ui-charts/src/highcharts/chartTypes/waterfallChart/tests/waterfallChartOptions.test.ts (hunk 4 above)
Comment (ivanmjartan) [reply libs/sdk-ui-charts/src/highcharts/chartTypes/waterfallChart/tests/waterfallChartOptions.test.ts:127]: waterfallChart was not possible create on goodsales same way some cases not nested now

File: tools/mock-handling/src/index.ts (hunk 18 above)
Comment (ivanmjartan) [reply tools/mock-handling/src/index.ts:35]: in internal tooling is only allowed tiger

File: tools/mock-handling/src/recordings/common.ts (hunk 19 above)
Comment (ivanmjartan) [reply tools/mock-handling/src/recordings/common.ts:89]: removed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18464/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20105/ : SUCCESS in 10m 36s

File: libs/sdk-backend-mockingbird/src/recordedBackend/tests/execution.test.ts (hunk 3 above)
Comment (kandl) [reply libs/sdk-backend-mockingbird/src/recordedBackend/tests/execution.test.ts:56]: `isIdRef` does not do the trick?

File: libs/sdk-backend-mockingbird/src/recordedBackend/tests/execution.test.ts (hunk 2 above)
Comment (ivanmjartan) [reply libs/sdk-backend-mockingbird/src/recordedBackend/tests/execution.test.ts:32]: fixed

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18466/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20108/ : FAILURE in 6m 07s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18467/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20109/ : FAILURE in 10m 50s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18469/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20111/ : FAILURE in 10m 58s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18470/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20112/ : SUCCESS in 12m 51s

File: tools/mock-handling/tsconfig.json [hunk 24]
```diff
@@ -13,8 +13,8 @@
         "incremental": false,
         "resolveJsonModule": true,
         "strict": true,
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
+        "noUnusedLocals": false,
```
Comment (ivanmjartan) [inline tools/mock-handling/tsconfig.json:16]: put it back

File: tools/mock-handling/src/recordings/dashboards.ts (hunk 20 above)
Comment (ivanmjartan) [reply tools/mock-handling/src/recordings/dashboards.ts:81]: We have git right ?

File: libs/sdk-backend-mockingbird/api/sdk-backend-mockingbird.api.md (hunk 17 above)
Comment (ivanmjartan) [reply libs/sdk-backend-mockingbird/api/sdk-backend-mockingbird.api.md:74]: We have git right ?

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR4811

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/4094/ : SUCCESS in 14m 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/1973/ : SUCCESS in 2m 30s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/20116/ : SUCCESS in 10m 56s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/1762/ : SUCCESS in 11m 43s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/1746/ : SUCCESS in 12m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/5269/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-compatibility-pipeline/683/ : SUCCESS in 9m 05s

**Pull request merged**